### PR TITLE
readme and script updates

### PR DESCRIPTION
**Pull request by ckhened**

### CODE REVIEW

File: ChatQnA/serving/tgi_gaudi/launch_tgi_service.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
 # Set default values
 default_port=8080
-default_model="Intel/neural-chat-7b-v3-3"
-default_num_cards=1
+default_model="meta-llama/Llama-2-70b-chat-hf"
```
Comment (hshen14) [inline ChatQnA/serving/tgi_gaudi/launch_tgi_service.sh:19]: llama2 has license issue, so we are not allowed to use this. please change back to the original one.

File: ChatQnA/README.md [hunk 2]
```diff
@@ -233,28 +273,3 @@ curl 127.0.0.1:8080/generate \
   -d '{"inputs":"What is Deep Learning?","parameters":{"max_new_tokens":32}}' \
   -H 'Content-Type: application/json'
 ```
-
```
Comment (lvliang-intel) [inline ChatQnA/README.md:236]: Why remove TEI part?

File: README.md [hunk 3]
```diff
@@ -10,6 +10,7 @@ This project provides a collective list of Generative AI (GenAI) examples such a
 ---
 
 <div align="left">
+This project provides a collective list of Generative AI (GenAI) and RAG (Retrieval and Augmentation) examples such as chatbot with question and answering (ChatQnA), code generation, document summary, etc. The examples are developed by leveraging the ecosystem components such as LangChain with Intel optimizations, therefore delivering the high performance for your deployment on Intel platforms.
```
Comment (lvliang-intel) [inline README.md:13]: RAG (Retrieval and Augmentation) should be RAG (Retrieval Augmentation Generation)

File: ChatQnA/README.md [hunk 4]
```diff
@@ -73,13 +88,44 @@ You have the flexibility to customize these parameters according to your specifi
...
+```
+
+## Launch LangChain Docker
+
+Update the `HUGGINGFACEHUB_API_TOKEN` environment variable with your huggingface token in the `docker-compose-langchain.yml`
 
 ```bash
 cd langchain/docker
-docker compose -f docker-compose.yml up -d
+docker compose -f docker-compose-langchain.yml up -d
```
Comment (lvliang-intel) [inline ChatQnA/README.md:128]: docker-compose-langchain.yml is renamed to docker-compose.yml

File: ChatQnA/README.md [hunk 5]
```diff
@@ -73,7 +88,38 @@ You have the flexibility to customize these parameters according to your specifi
...
+    -d '{"inputs":"What is Deep Learning?"}' \
+    -H 'Content-Type: application/json'
+```
+
+Note: If you want to integrate the TEI service into the LangChain application, you'll need to restart the LangChain backend service after launching the TEI service.
+
+## Launch Redis
+
+```bash
+docker compose -f langchain/docker/docker-compose-redis.yml up -d
```
Comment (chensuyue) [inline ChatQnA/README.md:119]: This file didn't existed anymore.   
We combine Launch Redis and LangChain Backend Service into one yml.

Comment (chensuyue) [reply ChatQnA/README.md:119]: Merge first, will fix in another PR.

Comment (jitendra42) [inline ChatQnA/README.md:119]: Change command to launch the redis: 
`docker compose -f langchain/docker/docker-compose.yml up -d redis-vector-db`

**Review by jitendra42 — COMMENTED** [review]
Assuming another PR will be created quickly to fix instructions

**Pull request merged**